extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -59,6 +60,18 @@ enum SortOrder {
   DateAddedDesc = "DateAddedDesc",
 }
 
+enum LanguageFilter {
+  All = "All",
+  Cpp = QueryLanguage.Cpp,
+  CSharp = QueryLanguage.CSharp,
+  Go = QueryLanguage.Go,
+  Java = QueryLanguage.Java,
+  Javascript = QueryLanguage.Javascript,
+  Python = QueryLanguage.Python,
+  Ruby = QueryLanguage.Ruby,
+  Swift = QueryLanguage.Swift,
+}
```

Comment:
It would be nice to avoid duplicating this language list again, but I wasn't sure how to do that 🙈

Reply:
I think (from a quick skim on StackOverflow) you can do something like
```ts
type LanguageFilter = QueryLanguage | 'All'
```
to reuse the constants, then you have to reference the constants in the code as either `QueryLanguage.<language>` or `All`.